crates/forge/tests/e2e/diagnostics_and_plugins.rs

```diff
@@ -13,32 +13,32 @@ fn print_error_if_attributes_incorrect() {
         output,
         indoc! {r#"
             error: Plugin diagnostic: Expected fork config must be of the form `url: <double quote string>, block_id: <snforge_std::BlockId>`.
-             --> lib.cairo:2:7
-            #[fork(url: "https://lib.com")]
-                  ^**********************^
+             --> [..]/src/lib.cairo:4:11
```

Comment:
Why are `[..]` needed here now? What will be this part of the path?

Reply:
Absolute path